paasta_tools/api/views/instance.py

```diff
@@ -400,3 +400,61 @@ def instance_mesh_status(
         raise ApiFailure(error_message, 500)
 
     return instance_mesh
+
+
+@view_config(
+    route_name="service.instance.replica.restart",
+    request_method="POST",
+    renderer="json",
+)
+def instance_replica_restart(
+    request: Request,
+) -> dict[str, Any]:
```

Comment:
non-blocking: i wonder if a typeddict or using the generated model would maybe be nicer here?